db/views/analytics_hmis_staff_assignments_v01.sql

```diff
@@ -0,0 +1,13 @@
+SELECT
+    id,
+    user_id,
+    household_id,
+    hmis_staff_assignment_relationship_id,
```

Comment:
It seems maybe overly complicated to send out the `hmis_staff_assignment_relationships` table to superset/dbt. It is a small configuration table specifying which types of relationships are available in the installation. I think this would be a case where it may make sense to do some pre-processing in the view? Do you agree?

My ActiveRecord version of this is: `Hmis::StaffAssignment.joins(:staff_assignment_relationship).select(:id, :user_id, :household_id, Hmis::StaffAssignmentRelationship.arel_table[:name], :data_source_id, :created_at, :updated_at).to_sql`


```suggestion
    "hmis_staff_assignment_relationships"."name",
```

Reply:
seems like a good simplification, thanks!